devcontainer/template/.devcontainer/general-devcontainer/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# syntax=docker/dockerfile:1
+FROM ghcr.io/everest/everest-ci/dev-env-base:v1.3.2
+
+# Update the package list
+RUN sudo apt update
```

Comment:
sudo should not be needed here

Reply:
It is necessary, because the image is based on `dev-env-base` image which swithces user [here](https://github.com/EVerest/everest-ci/blob/376eb2838eedf127ecdfd0b4e5414e6c247b72a7/docker/images/dev-env-base/Dockerfile#L100)